IBM/carbon-addons-iot-react
# 1130 combobox

Closes #1264 closes #1265 

**Summary**

Extend the Carbon Combobox component to implement dynamic input and multi-value input

**Change List (commits, features, bugs, etc)**

- Duplicated Carbon's combobox component in CarbonCombobox.jsx. This will be removed once we have migrated to `carbon-components-react@7.13.0`
- Created our own ComboBox Component in ComboBox.jsx
- Created test for both ComboBox.js and CarbonComboBox.jsx
- added styles to _combo-box.scss

**Acceptance Test (how to verify the PR)**

- Go to the experimental Combobox story and play with each one. The existing stories should work the same as base carbon with the addition of adding the new item to the dropdown list
- Go to multi-value story and make sure tags are added and removed  properly.
- Actions pannel should show the `onChange` callback being called with the appropriate value
